src/CONST.js

```diff
@@ -5,6 +5,7 @@ const CONST = {
     PDF_VIEWER_URL: '/pdf/web/viewer.html',
     EXPENSIFY_ICON_URL: `${CLOUDFRONT_URL}/images/favicon-2019.png`,
     UPWORK_URL: 'https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&user_location_match=2',
+    SMS_DOMAIN: 'expensify.sms',
```

Comment:
NAB, feel like this is already a const in `expensify-common`. Should we import it from there instead of creating a new one here?